canonical-webteam/unified-email-template
# Update the headers with the new design

## Done
Updated the newsletter headers to match the latest designs.

## QA
- Check /examples/product-branded.html
- Check /examples/insights-branded.html
- Check /examples/ubuntu-branded.html
- Check /examples/canonical-branded.html
- Check they match the design in the issue (be kind, its table layout) 

Fixes https://github.com/canonical-webteam/unified-email-template/issues/1
